pkg/config/database.go

```diff
@@ -18,7 +22,10 @@ import (
 	"time"
 )
 
-var registerDriverOnce sync.Once
+var (
+	registerDriverOnce sync.Once
+	wsrepSyncWait      int64
```

Comment:
Why is this a global variable? Wasn't the whole point of passing it in the DSN or using `NewConnector()` and wrapping the result into a `*sqlx.DB` to have this option scoped to the `*DB`?

Reply:
Do you mean scoped to `*config.Database`? Otherwise it doesn't make sense to add this to the actual `DB` type.

Reply:
I wanted to say so that it's possible to have multiple `*icingadb.DB` that could theoretically use different values for `wsrep_sync_wait`. For the Icinga DB daemon, this doesn't really mater, but you just have to look at the history migration tool where you have to configure an IDO connection as well using the same config, so already there you could write config that results in strange behavior if a global variable would be used.

And fixing this wouldn't even be hard with the current code. Instead of setting a global variable, just use an anonymous function for `InitConn` that captures the value.